### PR TITLE
Update documentation for PiGx-SARS-CoV-2

### PR DESCRIPTION
**Pull request by jonasfreimuth**
Will update the documentation on PiGx-SARS-CoV-2 to the state of the pipeline after PR BIMSBbioinfo/pigx_sars-cov-2#145 is merged.

### CODE REVIEW

**Review by rekado — APPROVED** [review]
I haven't read the changes to pigx-sars-cov-2.md, but the other changes look fine to me.

**Comment by jonasfreimuth** [comment]
See [this](https://github.com/BIMSBbioinfo/pigx_sars-cov-2/pull/145#pullrequestreview-1127082283) conversation for the changes to the Quick start section.

File: pages/pigx-sars-cov-2.md [hunk 1]
```diff
@@ -79,51 +134,80 @@ make
...
+take some time.
 
-We provide the script `download_databases.sh` to automate the download
-and configuration of all default databases.  After installing the
-pipeline you can run the script like this:
+Without any user intervention, this will happen automatically via snakemake
+rules. This behaviour is controlled via parameters in the settings file. See the
+[Settings file](#settings-file) section for details.
+
+Alternatively, the databases may be downloaded maunally via the
```
Comment (rekado) [inline pages/pigx-sars-cov-2.md]: typo: maunally --> manually

Comment (jonasfreimuth) [reply pages/pigx-sars-cov-2.md]: Fixed ^^

**Pull request merged**